sources/niLang/src/DataTablePath.h

```diff
@@ -56,10 +55,12 @@ enum eDataTablePathOp
   eDataTablePathOp_Parent = 2,
   //! Path operation, datatable.
   eDataTablePathOp_DataTable = 3,
+  //! Path operation, child datatable.
+  eDataTablePathOp_ChildDataTable = 4,
```

Comment:
Just put ChildDataTable after Property so that we dont have a diff here. In general we should preserve the values of enums unless there's really no other way.

Reply:
We ended up deciding to change for _Predicate using '[]' instead.